packages/vizbuilder/src/helpers/chartconfig.js

```diff
@@ -383,8 +385,12 @@ export default function createChartConfig(
     value: getMeasureValue
   };
 
-  if (hasTimeDim) {
+  if (hasTimeDim && activeType) {
     commonConfig.time = timeLevelName;
+    commonConfig.timeline = true;
+  } 
+  else {
+    commonConfig.timeline = false;
```

Comment:
we still need to set `commonConfig.time = timeLevelName;` for non-active visualizations (it sometimes effects formatting or axes)